clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -139,7 +139,7 @@ public class TopicConfig {
     public static final String CLEANUP_POLICY_CONFIG = "cleanup.policy";
     public static final String CLEANUP_POLICY_COMPACT = "compact";
     public static final String CLEANUP_POLICY_DELETE = "delete";
-    public static final String CLEANUP_POLICY_DOC = "A string that is either \"" + CLEANUP_POLICY_DELETE +
+    public static final String CLEANUP_POLICY_DOC = "A comma separated list of String(s) that is either \"" + CLEANUP_POLICY_DELETE +
```

Comment:
This config could use a rewrite. Let me take a stab.

> This config designates the retention policy to use on old log segments. The "delete" policy (which is the default) will discard old segments when their retention time or size limit has been reached. The "compact" policy will enable log compaction. It is also possible to specify both policies in a comma-separated list (e.g. "delete,compact"). In this case, old segments will be discarded per the retention time and size configuration, while retained segments will be compacted.

What do you think?

Reply:
Thank you @hachikuji , this is indeed way better !